packages/documentation-site/patternfly-docs/content/developer-resources/rtl-handbook.md

```diff
@@ -33,9 +33,11 @@ Care should be taken to handle icons appropriately for RTL. Some icons you may h
 
 ## PatternFly RTL helpers
 
-To reverse an icon or other element when in RTL, use the modifier class `.pf-v5-m-mirror-inline-rtl` or the `shouldMirrorRTL` property of the `<Icon>` React component.
+The preferred approach for defining an element's direction is to use the `dir` HTML attribute: `dir="rtl"` or `dir="ltr"`. If you are unable to use this attribute for any reason, you can instead utilize one of the following approaches to reverse an icon or other element when in RTL:
 
-To set direction manually on an element, use `.pf-v5-m-dir-ltr` or `.pf-v5-m-dir-rtl`.
+- Use the `.pf-v5-m-mirror-inline-rtl` class
+- Manually set an element's direction with `.pf-v5-m-dir-ltr` or `.pf-v5-m-dir-rtl`
+- Use the `shouldMirrorRTL` property of the `<Icon>` React component
```

Comment:
The way this reads, I wonder if it's conflating a couple of ideas. I could be reading it wrong though 😅 So this may be known, but there is a difference between:
1. `dir`, `.pf-v5-m-dir-ltr`, and `.pf-v5-m-dir-rtl`
  * This changes the "writing direction", and can apply to the whole document or individual elements, and can be nested (the document can be RTL, a block within the document can be LTR, and a block within the LTR block can be RTL, and so on). It basically just tells the browser the writing mode is from right to left. It will attempt to change the order of things to start from the right and end at the left. It's a browser feature with a bunch of logical stuff that happens to try and accommodate an LTR language. Nothing will happen with an icon, it's more for the way content and boxes are laid out.
    * `dir` should be used when the writing mode of a thing needs to be set. The `.pf-v5-m-dir[...]` classes will change the writing direction, but only visually/presentationally.
3. `.pf-v5-m-mirror-inline-rtl` and `<Icon shouldMirrorRTL />`
  * They just flip something horizontally. They don't really have anything to do with RTL/LTR per se, aside from these helpers have been written to only apply when they're used in an RTL block.

Reply:
no you're right! I was definitely understanding this differently. I think I'm _mostly_ following, so I just pushed some changes that hopefully incorporate some of this distinction! Pls keep correcting the content as needed though 🙏